services/github/github-api-provider.integration.js

```diff
@@ -79,7 +79,7 @@ describe('Github API provider', function () {
       }
     })
 
-    it.skip('should update the token with the final limit remaining and reset time', function () {
+    it('should update the token with the final limit remaining and reset time', function () {
```

Comment:
Is it worth including this link here, for the next time it fails on someone?

```suggestion
    // Is this test failing? See https://github.com/badges/shields/pull/4590#issuecomment-708551801
    it('should update the token with the final limit remaining and reset time', function () {
```